content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -0,0 +1,173 @@
+
+1. Configure `rippled.cfg`.
+
+    `rippled.cfg` is the configuration file for `rippled`.
+
+    You can find the `rippled-example.cfg` example file in `rippled/doc`. See the file for a description of all configuration options. You should be able to connect to the XRP Ledger using the default configuration defined in the example file.
+
+    Put your configured `rippled.cfg` file in `etc/opt/ripple/`.
+
+2. Configure `validators.txt`.
+
+    When you start your `rippled` server for the first time, it uses an initial list of validators defined in `validators.txt` to retrieve validation information and to form its UNL (unique node list). The information in `validators.txt` also enables `rippled` clients to indirectly locate IPs they can use to contact the Ripple network.
+
+    You can find the `validators-example.txt` example file in `rippled/doc`. See the file for a description of all validator options. You should be able to connect to the XRP Ledger using the default validators defined in the example file.
+
+    Put your configured `validators.txt` file in `etc/opt/ripple/`.
+
+
+## 3. Run `rippled`
+
+To run your stock `rippled` server from the executable you built, using the configurations you defined:
+```
+cd build
+sudo ./rippled
+```
+
+
+### What to Expect
+
+Once you've run `rippled`, here are excerpts of what you can expect to see in your terminal:
+
+```
+Loading: "/etc/opt/ripple/rippled.cfg"
+Watchdog: Launching child 1
+2017-Dec-15 18:27:37 JobQueue:NFO Auto-tuning to 4 validation/transaction/proposal threads for non-validator.
+2017-Dec-15 18:27:37 Amendments:DBG Amendment C6970A8B603D8778783B61C0D445C23D1633CCFAEF0D43E7DBCD1521D34BD7C3 is supported.
+2017-Dec-15 18:27:37 Amendments:DBG Amendment 4C97EBA926031A7CF7D7B36FDE3ED66DDA5421192D63DE53FFB46E43B9DC8373 is supported.
+...
+2017-Dec-15 18:27:37 ValidatorList:DBG Loading configured trusted validator list publisher keys
+...
+2017-Dec-15 18:27:37 ValidatorList:DBG Loading configured validator keys
+...
+2017-Dec-15 18:27:37 ValidatorSite:DBG Loading configured validator list sites
+...
+2017-Dec-15 18:27:38 LedgerConsensus:NFO Entering consensus process, watching, synced=no
+...
+2017-Dec-15 18:27:40 LedgerConsensus:WRN
+{
+	"accepted" : true,
+	"account_hash" : "183D5235C7C1FB5AE67AD2F6CC3B28F5FB86E8C4F89DB50DD85641A96470534E",
+	"close_flags" : 0,
+	"close_time" : 566677650,
+	"close_time_human" : "2017-Dec-15 18:27:30",
+	"close_time_resolution" : 30,
+	"closed" : true,
+	"hash" : "014816C71F4B9AD5924AFBD7B02E0DF1522C4BC4BC45A69F97F51D4F2234FA10",
+	"ledger_hash" : "014816C71F4B9AD5924AFBD7B02E0DF1522C4BC4BC45A69F97F51D4F2234FA10",
+	"ledger_index" : "2",
+	"parent_close_time" : 0,
+	"parent_hash" : "AB868A6CFEEC779C2FF845C0AF00A642259986AF40C01976A7F842B6918936C7",
+	"seqNum" : "2",
+	"totalCoins" : "100000000000000000",
+	"total_coins" : "100000000000000000",
+	"transaction_hash" : "0000000000000000000000000000000000000000000000000000000000000000"
+}
+...
```

Comment:
It would be great to provide a description of each of the following: 

* NetworkOPs: WRN We are not running on the consensus ledger. Who is not running on the consensus ledger and what does this mean?
* Ledgerconsensus:WRN Need consensus ledger. Who need it and why?
* OpenLedger:WRN Offer involves frozen asset. What does this mean?
* LedgerConsensus:WRN Offer involves frozen asset. What does this mean?